lms/static/js/learner_dashboard/spec/course_card_view_spec.js

```diff
@@ -13,8 +13,13 @@ describe('Course Card View', () => {
     const setupView = (data, isEnrolled, collectionCourseStatus) => {
         const programData = $.extend({}, data);
         const context = {
-            courseData: {},
+            programData,
             collectionCourseStatus,
+            courseData: {},
+            subscriptionData: [],
+            urls: {},
+            userPreferences: {},
+            isSubscriptionEligible: false,
```

Comment:
If I'm following the logic correctly, other than `url`, the rest of this context isn't used in `CourseCardView`. Is there a plan to use them in future iterations, or am I missing the reason to include them? @NawfalAhmed

Reply:
Ah, nevermind, I was going down a different rabbit hole 😅 all good here.